pkg/crypto/api.go

```diff
@@ -62,6 +60,28 @@ type Crypto interface {
 	// 		unwrapped key in raw bytes
 	// 		error in case of errors
 	UnwrapKey(recWK *RecipientWrappedKey, kh interface{}, opts ...WrapKeyOpts) ([]byte, error)
+	// BBSSign will sign create signature of each message in messages and aggregate it into a single signature using a
+	// matching signature primitive in kh key handle of a private key.
+	// returns:
+	// 		signature in []byte
+	//		error in case of errors
+	BBSSign(messages [][]byte, kh interface{}) ([]byte, error)
```

Comment:
It’s weird to call this BBS - make a generic name in an interface.

Reply:
Sign() is already used

Reply:
updated to SignMulti()